ezsystems/ezplatform-admin-ui/CVE-2019-12139.yaml

```diff
@@ -0,0 +1,10 @@
+title:     EZSA-2019-001: XSS in Admin UI
```

Comment:
You need to quote the title as there is a colon inside (same for the other one).